doc/cephadm/install.rst

```diff
@@ -391,46 +391,45 @@ referred to as "airgapped".
 Ensure your container image is inside the registry and that you
```

Comment:
zdover23 - Two problems for a subsequent PR: (1) in line 387, _This_ needs to be clarified, so that the reader knows it refers to the second option ("make the registry secure"); (2) in line 386, the reader will initially and wrongly think that _either_ pairs with the _or_ in "Podman or Docker".